public/video-ui/src/slices/targeting.ts

```diff
@@ -0,0 +1,68 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+
+type TargetType={
+  id: string,
+  title: string,
+  tagPaths: string[],
+  url: string,
+  activeUntil?: number,
+  createdBy?: string,
+  updatedBy?: string,
+  createdAt?: string,
+  updatedAt?: string,
+}
+type TargetingState = {
+  targets: TargetType[]| null,
+  deleting: string[],
+};
+
+const initialState:TargetingState = { targets: null, deleting: [] };
+const targeting = createSlice({
+  name: 'targeting',
+  initialState,
+  reducers: {
+    requestUpdateTarget:(state:TargetingState, action :PayloadAction<TargetType>)=> ({
+        ...state,
+        targets: [
+          ...(state.targets || []).filter(({ id }) => id !== action.payload.id),
+          action.payload
+        ]
+    }),
+    receiveUpdateTarget:(state:TargetingState)=> ({
```

Comment:
This action and the one below `requestCreateTarget` do not change the state at all.  Do you think if it's worth removing them to simplify the reducer?